mappings/1a899100ce60be94398ce171d6e99daa4aba757c64e9faedf13256d242494553.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "1a899100ce60be94398ce171d6e99daa4aba757c64e9faedf13256d242494553",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://bitearthswap.com",
```

Comment:
Hi @BitearthSwap,

can you please put policyID of this token on your website, so we can verify it?

Thanks in advance!

Reply:
Hi @VladislavKudrin , thanks for the update.

The website has been updated with the policy ID, Kindly verify

Thanks in advance